.baseline/checkstyle/checkstyle-suppressions.xml

```diff
@@ -21,7 +25,5 @@
     <suppress files="\.(bdr|eot|gif|gzip|jar|json|otf|png|svg|ttf|woff|zip)$" checks="NewlineAtEndOfFile" />
 
     <!-- Generated code should not be subjected to checkstyle. -->
-    <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
-    <suppress files="[/\\].*[/\\]generated_testSrc[/\\]" checks="." />
-    <suppress files="[/\\].*[/\\]generated[/\\]" checks="." />
```

Comment:
For next time, running `./gradlew baselineUpdateConfig` with updated gradle-baseline plugin should generate these checkstyle config changes automatically

Reply:
Ah, awesome. Thanks :)